File: src/vlab/server_js/check/PostCheckProcessorImpl.js
function postCheck(preCheckResult, checkingResults, checkers){

    print(preCheckResult['value']);

    checkingResults.forEach(function(checkingResult){
        print('id: ' + checkingResult['id']);
        print('time: ' + checkingResult['time']);
        print('result: ' + checkingResult['result']);
        print('output: ' + checkingResult['output']);
    });

    checkers.forEach(function(checkProcessor){
        //????
        print(checkProcessor);
    });

}

File: src/vlab/server_java/check/PostCheckProcessorImpl.java
package vlab.server_java.check;

import rlcp.check.CheckingResult;
import rlcp.server.processor.check.CheckProcessor;
import rlcp.server.processor.check.PostCheckProcessor;
import rlcp.server.processor.check.PreCheckProcessor;

import java.util.List;

import static rlcp.server.processor.check.PreCheckProcessor.*;

/**
 * Simple PostCheckProcessor implementation. Supposed to be changed as needed to provide
 * necessary after checking support.
 */
public class PostCheckProcessorImpl implements PostCheckProcessor{
    @Override
    public void postCheck(PreCheckResult preCheckResult, List<CheckingResult> checkingResults, List<CheckProcessor> checkers) {

    }
}
